webapp/packages/core-blocks/src/FormControls/InputAutocompletionMenu.tsx

```diff
@@ -0,0 +1,81 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+import { observer } from 'mobx-react-lite';
+import type { RefObject } from 'react';
+
+import BaseDropdownStyles from '../FormControls/BaseDropdown.module.css';
+import { getComputed } from '../getComputed.js';
+import { IconOrImage } from '../IconOrImage.js';
+import { Menu } from '../Menu/Menu.js';
+import { MenuItem } from '../Menu/MenuItem.js';
+import type { IMenuState } from '../Menu/MenuStateContext.js';
+import { s } from '../s.js';
+import { Text } from '../Text.js';
+import { useS } from '../useS.js';
+import style from './InputAutocompletionMenu.module.css';
+import { type InputAutocompleteProposal } from './useInputAutocomplete.js';
+
+interface AutocompletionProps {
+  position: { x: number; y: number };
+  proposals: InputAutocompleteProposal[];
+  menuRef: RefObject<IMenuState>;
+  className?: string;
+  onSelect?: (proposal: InputAutocompleteProposal) => void;
+}
+
+export const InputAutocompletionMenu = observer(function InputAutocompletionMenu({
+  position,
+  className,
+  menuRef,
+  proposals,
+  onSelect,
+}: AutocompletionProps) {
+  const styles = useS(style, BaseDropdownStyles);
+  const contextMenuPosition = getComputed(() => ({
+    position,
+    handleContextMenuOpen: () => {},
+  }));
```

Comment:
getComputed will not help cuz function always returns new object (getComputed works if you can return stable value, for example for primitives)